expense-frontend/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:20 as build
+
+WORKDIR /App-Frontend
```

Comment:
Choose simple names for simple stuffs. `/App-Frontend` is complex because it contains a special character, `-`. You can go for `/app` or `/frontend`.